lib/fetchers/fetcher.py

```diff
@@ -18,12 +16,16 @@
 from dplaingestion.selector import setprop
 from dplaingestion.selector import getprop as get_prop
 from dplaingestion.utilities import iterify, couch_id_builder
+import requests
+import re
```

Comment:
@markbreedlove it is used downstream. Removing it here breaks other classes so I just added it back rather than adding it in every other location. This cascading include issue is a bit of a pain.